lib/views/help/ico_officers.html.erb

```diff
@@ -0,0 +1,128 @@
+      <p>
+      With hundreds of requests processed by WhatDoTheyKnow every day, we don’t check the contents of every single one — but under our policy of <a href="/help/how">reactive moderation</a>, when we notice that an authority has accidentally released sensitive information or someone reports a serious data breach to us, we take it offline as soon as possible and report it to the ICO or OSIC where applicable.
+      </p>
+      <p>
+      In such cases, we are happy to help you understand what has happened and to provide any further details you require to help you handle the report. Note that we are neither the subject of the data breach nor the authority that released the sensitive data in such cases.
+    </p>
+    <p>
+      We try to help public bodies improve their processes so that accidental releases like this don’t happen, and, abiding by the ICO’s own guidance on when to do so, we report cases to you if applicable.
+    </p>
+
+    </dd>
+    <dt id="large_data_transfers">
+      I need you to transfer a large amount of personal data securely, for my investigations into a data breach
+      <a href="#large_data_transfers">#</a>
+    </dt>
+    <dd>
+      <p>
+      Where appropriate, we can provide data to aid with your investigations. Before we do so, we require details of how it will be used and why it is needed, which will be considered according to our policies.
+      </p>
+      <p>
+      We have procedures to ensure that such data is transferred securely. Please <a href="/help/contact">get in touch</a> with us to discuss details.
+      </p>
+    </dd>
+    <dt id="requests_over_time">
+      I am trying to understand whether an authority is meeting its duties under the law
+      <a href="#requests_over_time">#</a>
+    </dt>
+    <dd>
+      <p>
+      Because requests and responses are published openly on the site, WhatDoTheyKnow can be a useful tool for assessing how authorities are dealing with requests over time. Please do <a href="/help/contact">get in touch</a> if you would like to discuss this further.
+      </p>
+    
+    </dd>
+
+    <dt id="useful_facts">
+    Some facts it’s useful for you to know
+      <a href="#useful_facts">#</a>
+    </dt>
+    <dd>
+      <ul>
+        <li>
+        The process of publishing FOI responses on WhatDoTheyKnow is automatic. There is no human involvement, but we do promptly consider any concerns to which our attention is drawn.
+        </li>
+        <li>
+        Requests are made by our users, not by the WhatDoTheyKnow.com admin team.
+        </li>
+        <li>
+        If a request isn’t visible (say, you’ve clicked a link but it goes to a ‘page not found’), it may have been removed. This can be due to one of a number of reasons, including the removal of requests that are vexatious, contain personal information or are not FOI requests. Some requests, sent through our <a href="/pro">Pro</a> service, may be hidden from the public until an embargo has expired.
```

Comment:
```suggestion
        If a request isn’t visible (say, you’ve clicked a link but it goes to a ‘page not found’), it may have been removed. This can be due to one of a number of reasons, including the removal of requests that are vexatious, contain personal information or are not FOI requests. Some requests, sent through our <%= link_to 'Pro', account_request_index_path %> service, may be hidden from the public until an embargo has expired.
```